Doc/library/multiprocessing.rst

```diff
@@ -1942,6 +1947,14 @@ their parent process exits.  The manager classes are defined in the
 
       Create a shared :class:`list` object and return a proxy for it.
 
+   .. method:: set()
+               set(sequence)
+               set(mapping)
+
+      Create a shared :class:`set` object and return a proxy for it.
+
+      .. versionadded:: next
```

Comment:
```suggestion
      .. versionadded:: next
         :class:`set` support was added.
```
its nice to include text to help disambiguate the versionadded/changed notes in the docs.  otherwise it isn't always clear if the marker is tied to one thing or a whole section of things.